components/BaseModal.vue

```diff
@@ -5,39 +5,39 @@ const emit = defineEmits<{
   (e: "closeModal"): void;
 }>();
 
-const focusableNodes = computed(() => {
-  return root.value?.querySelectorAll<HTMLInputElement>(
-    "a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type='text']:not([disabled]), input[type='radio']:not([disabled]), input[type='checkbox']:not([disabled]), select:not([disabled])"
-  );
-});
+const focusableNodes = () => {
+  if (root.value) {
+    return root.value?.querySelectorAll<HTMLElement>(
+      "a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type='text']:not([disabled]), input[type='radio']:not([disabled]), input[type='checkbox']:not([disabled]), select:not([disabled])"
+    );
+  }
+};
 
 const firstFocusableNode = () => {
-  if (focusableNodes.value) return focusableNodes.value[0];
+  if (focusableNodes() !== undefined) {
+    return focusableNodes()[0];
+  }
 };
```

Comment:
Here I have a problem with typescript. FocusableNodes can return `focusableNodes: () => NodeListOf<HTMLElement> | undefined`, so in the firstFocusableNodes function I check if focusableNodes() is not undefined. But for for `focusableNodes()[0];` I still get typescript error: Object is possibly 'undefined'.ts(2532).
What should I do here to get rid of the error?, any hints?